config/jobs/kubernetes-sigs/cluster-api/cluster-api-periodics-main.yaml

```
@@ -188,7 +188,7 @@ periodics:
       # This value determines the minimum Kubernetes
       # supported version for Cluster API management cluster.
       - name: KUBERNETES_VERSION_MANAGEMENT
-        value: "v1.24.15"
+        value: "v1.25.12"
```

```suggestion
        value: "v1.25.11"
```
I think this should be a version for which a kindest/node image exists. Otherwise we have to build the image on every run.

I also think that ideally we pin this to a specific sha of our current kind version (https://github.com/kubernetes-sigs/kind/releases/tag/v0.20.0)

Let's also add a godoc comment here

@killianmuldoon WDYT?

---

Won't the mapper do the pin handling for us?

---

As far as I know the mapper is not used where this variable is used (setup of the bootstrap cluster)

---

If I'm not missing anything we use a sha for the fallback value: https://github.com/kubernetes-sigs/cluster-api/blob/6a01c0edc502b60ac5a0333d7c5970b4afa6032e/test/framework/bootstrap/kind_provider.go#L40
(which is never used in core CAPI)

---

Jap, just also did take a look into the code 👍